recipe/meta.yaml

```diff
@@ -40,7 +40,7 @@ test:
     - pip check
 
```

Comment:
Should we try `pytest tests` as the package becomes more and more complicated? See https://github.com/fastapi/fastapi/blob/0.112.2/requirements-tests.txt#L3. I'm not sure yet which test dependencies also needed.

Reply:
It was attempted. There are still 6 or more dependencies that are not satisfied in the defaults channel to run the test suite.

Reply:
A small note that running those tests will probably be painful in our current infrastructure.